modules/internal/ChapelStandard.chpl

```diff
@@ -24,25 +24,20 @@ module ChapelStandard {
   use startInitCommDiags;  // Internal, but uses standard/CommDiagnostics
 
   // Internal modules.
-  use CPtr;
   use CString;
   use Bytes;
   use String;
   use OwnedObject;
   use SharedObject;
-  use ChapelDebugPrint;
   use ChapelEnv;
   use ChapelBase;
-  use MemConsistency;
```

Comment:
This is what makes `memory_order` availble. I think this needs to be available to users too.

Reply:
I think it still is available to users due to the public `use` of it via `Atomics.chpl`?

My ideal (that this could be considered a step towards... and I considered adding comments to ChapelStandard to help with it) would be that each `use` within ChapelStandard could be mapped directly to a language feature.  So for example `use Atomics` would provide everything that's necessary to support `atomic t` in the language.  In this first pass, I tried to prune stuff that didn't seem obviously related to the language (at least at that first-class level of features) to see what would break.

Reply:
Gotcha, makes sense